src/test/java/uk/gov/hmcts/divorce/citizen/notification/Applicant2ApprovedNotificationTest.java

```diff
@@ -237,5 +239,26 @@ void shouldSendEmailToApplicant1SolicitorWithDissolutionContent() {
             eq(ENGLISH)
         );
     }
+
+    @Test
+    void shouldSendEmailInWelshLanguageToApplicant1WhenRespondentApprovedAndLanguagePreferenceIsWelsh() {
+        CaseData data = validApplicant2CaseData();
+        data.setDivorceOrDissolution(DIVORCE);
+        data.getApplication().getApplicant1HelpWithFees().setNeedHelp(YesOrNo.YES);
+        data.getApplication().getApplicant2HelpWithFees().setNeedHelp(YesOrNo.YES);
+        data.setDueDate(LOCAL_DATE);
+        data.getApplicant1().setLanguagePreferenceWelsh(YesOrNo.YES);
+
+        notification.sendToApplicant1(data, 1234567890123456L);
+
+        verify(notificationService).sendEmail(
+            eq(TEST_USER_EMAIL),
+            eq(JOINT_APPLICANT1_APPLICANT2_APPROVED),
+            anyMap(),
```

Comment:
for welsh I would also add check something like https://github.com/hmcts/nfdiv-case-api/pull/2003/files#diff-6628fc0f1cc3ddd9bf790381b31bd2a9b892cdd24889fe4feb503a3e08b3720cR137

Reply:
done